code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1630,3 +1630,14 @@
 	description = "blue sparkles that get everywhere"
 	color = "#4040FF" //A blueish color
 	glitter_type = /obj/effect/decal/cleanable/glitter/blue
+
+/datum/reagent/pukonium
```

Comment:
I wasn't sure if that was allowed by the feature freeze as it's a pretty antag-ish reagent.